packages/twenty-front/src/modules/object-record/record-filter/utils/turnObjectDropdownFilterIntoQueryFilter.ts

```diff
@@ -24,6 +24,128 @@ export type ObjectDropdownFilter = Omit<Filter, 'definition'> & {
   };
 };
 
+const applyEmptyFilters = (
+  operand: ViewFilterOperand,
+  correspondingField: Pick<Field, 'id' | 'name'>,
+  objectRecordFilters: RecordGqlOperationFilter[],
+  filterType:
+    | 'URL'
+    | 'Address'
+    | 'String'
+    | 'Number'
+    | 'Date'
+    | 'Relation'
+    | 'CompositeString' = 'String',
+) => {
+  let emptyRecordFilter: RecordGqlOperationFilter = {};
+
+  switch (filterType) {
+    case 'String':
+      emptyRecordFilter = {
+        or: [
+          { [correspondingField.name]: { ilike: '' } as StringFilter },
+          { [correspondingField.name]: { is: 'NULL' } as StringFilter },
+        ],
+      };
+      break;
+    case 'CompositeString':
+      // eslint-disable-next-line no-case-declarations
+      const fullNameFilters = generateILikeFiltersForCompositeFields(
+        '',
+        correspondingField.name,
+        ['firstName', 'lastName'],
+      );
+      emptyRecordFilter = {
+        not: {
+          or: fullNameFilters,
+        },
+      };
+      break;
+    case 'URL':
+      emptyRecordFilter = {
+        or: [
+          { [correspondingField.name]: { url: { ilike: '' } } as URLFilter },
+          {
+            [correspondingField.name]: { url: { is: 'NULL' } } as URLFilter,
+          },
+        ],
+      };
+      break;
+    case 'Address':
+      emptyRecordFilter = {
+        or: [
+          {
+            [correspondingField.name]: {
+              addressStreet1: { ilike: '' },
+            } as AddressFilter,
+          },
+          {
+            [correspondingField.name]: {
+              addressStreet2: { ilike: '' },
+            } as AddressFilter,
+          },
+          {
+            [correspondingField.name]: {
+              addressCity: { ilike: '' },
+            } as AddressFilter,
+          },
+          {
+            [correspondingField.name]: {
+              addressStreet1: { is: 'NULL' },
+            } as AddressFilter,
+          },
+          {
+            [correspondingField.name]: {
+              addressStreet2: { is: 'NULL' },
+            } as AddressFilter,
+          },
+          {
+            [correspondingField.name]: {
+              addressCity: { is: 'NULL' },
+            } as AddressFilter,
+          },
+        ],
+      };
+      break;
+    case 'Number':
+      emptyRecordFilter = {
+        or: [
+          { [correspondingField.name]: { eq: 0 } as FloatFilter },
+          { [correspondingField.name]: { is: 'NULL' } as FloatFilter },
+        ],
+      };
+      break;
+    case 'Date':
+      emptyRecordFilter = {
+        or: [
+          { [correspondingField.name]: { eq: '' } as DateFilter },
+          { [correspondingField.name]: { is: 'NULL' } as DateFilter },
+        ],
+      };
+      break;
+    case 'Relation':
+      emptyRecordFilter = {
+        [correspondingField.name + 'Id']: { is: 'NULL' } as UUIDFilter,
```

Comment:
create a `RelationFilter` instead using `UUIDFilter`

Reply:
I think you should add one filter type dedicated to Relation even though it defines the same type as UUIDFilter. 
I would do: 
```
type RelationFilter = {
  is?: IsFilter;
}
```
This type can be enriched with other filter operations when necessary

Reply:
Thanks for clarifying, added the `RelationFilter` type